packages/components/src/icon/stories/index.js

```diff
@@ -1,128 +0,0 @@
-/**
- * External dependencies
- */
-import { number } from '@storybook/addon-knobs';
-
-/**
- * WordPress dependencies
- */
-import { SVG, Path } from '@wordpress/primitives';
-import { wordpress } from '@wordpress/icons';
-
-/**
- * Internal dependencies
- */
-import Icon from '../';
-import { VStack } from '../../v-stack';
-
-export default {
-	title: 'Components/Icon',
-	component: Icon,
-	parameters: {
-		knobs: { disable: false },
-	},
-};
-
-const IconSizeLabel = ( { size } ) => (
-	<div style={ { fontSize: 12 } }>{ size }px</div>
-);
-
-export const _default = () => {
-	const size = number( 'Size', '24' );
-
-	return (
-		<div>
-			<Icon icon={ wordpress } size={ size } />
-			<IconSizeLabel size={ size } />
-		</div>
-	);
-};
-
-export const sizes = () => {
-	const iconSizes = [ 14, 16, 20, 24, 28, 32, 40, 48, 56 ];
-
-	return (
-		<>
-			{ iconSizes.map( ( size ) => (
-				<div
-					key={ size }
-					style={ { padding: 20, display: 'inline-block' } }
-				>
-					<Icon icon={ wordpress } size={ size } />
-					<IconSizeLabel size={ size } />
-				</div>
-			) ) }
-		</>
-	);
-};
-
-export const colors = () => {
-	const iconColors = [ 'blue', 'purple', 'green' ];
```

Comment:
Also simplified this to only show one color, as this fits the Controls paradigm better.

Reply:
Theoretically, we could introduce a control for the color, but I'm not sure if that's worth the effort.